SCClassLibrary/Common/GUI/Base/Grid.sc

```diff
@@ -64,6 +64,14 @@ DrawGrid {
 	vertGrid_ { arg g;
 		y.grid = g;
 	}
+	tickSpacing_ { arg val;
+		x.tickSpacing = val;
+		y.tickSpacing = val;
```

Comment:
The thinking behind this one is that it's a convenience method. One can set individual tickSpacing with `~grid.x.tickSpacing_()`/`grid.y.tickSpacing_()` (assuming `~grid = DrawGrid(spec)`. However...

I realized that the view also needs to be updated once the tick spacing is reset. This only applies here to the `DrawGrid.test`, since otherwise the userview is outside of this class. In any case, I can see the following scenarios:
- keep `~grid.tickSpacing_` as a convenience, which updates the test view (if needed)
- remove `~grid.tickSpacing_` and have the user do `~grid.x.tickSpacing_`/`~grid.y.tickSpacing_` and then having to manually update the test view
- add `~grid.tickSpacingX_` and `~grid.tickSpacingY_` which updates respective spacing and updates the test view.

Or maybe updating the test view is not worth the hassle?

Reply:
Your addition of the `-testView` getter resolves this I think 👍
Could we add this example to the docs?
```
(
c = ControlSpec.new(20,200, \lin, units: "myUnits");
g = c.grid;
d = DrawGrid.test(g, g, 500@300);

// explicitly set the (minimum) number of x-axis ticks
d.y.numTicks_(5);     // fixed number of tix (approximate)
d.x.tickSpacing_(25); // fixed density of ticks
d.testView.refresh;   // refresh to update tick settings
// ... resize the view to observe the behavior
)
```